### PR TITLE
The docs have been moved from rhd-gitops-example.

### PR DESCRIPTION
**Pull request by gaganhegde**
**What type of PR is this?**
> /kind cleanup

**What does this PR do / why we need it**
The documentation has been moved from the rhd-gitops-example  to https://github.com/redhat-developer/gitops-cli .
Updated CLI reference to the current command line

**Have you updated the necessary documentation?**

See https://github.com/rhd-gitops-example/docs

-[x] Updated

**Which issue(s) this PR fixes**:
https://issues.redhat.com/browse/GITOPS-322

### CODE REVIEW

**Review by bigkevmcd — APPROVED** [review]

**Review by wtam2018 — CHANGES_REQUESTED** [review]
Let's make sure command reference are current (i.e. correctly documents each commands).   Also, we agreed to remove the sample output of manifests.

File: docs/commands/environment/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+| Flag                    | Description |
+| ----------------------- | ----------- |
+| --cluster               | Deployment cluster (Default https://kubernetes.local.svc)|
+| --env-name              | The name of environment to be added|
+| --pipelines-file        | Optional.  Path to manifest file.  Default is pipelines.yaml. |
+| --help                  | Show help|
+
+
+The following [directory layout](output) is generated.
```
Comment (wtam2018) [inline docs/commands/environment/README.md:24]: Please make sure all the references to the output are removed from the md files.

File: docs/journey/day1/README.md [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+You need to have the following installed in the OCP 4.x cluster.
+* [Sealed Secrets Operator](prerequisites/sealed_secrets.md)
+* [OpenShift Pipelines Operator](prerequisites/tekton_operator.md)
+* [ArgoCD](prerequisites/argocd.md)
+
+And, you will need this.
+* Create [GitOps repository](prerequisites/gitops_repo.md)
+* Source Git repository ([taxi](prerequisites/service_repo.md) is used as an example in this document)
+* The external image repository secret to authenticate image pushes on sucessfull pipeline execution. To use quay.io, please follow [prerequisites/quay.md](prerequisites/quay.md)
+* Download official [gitops](../../commands/bin) binary
```
Comment (wtam2018) [inline docs/journey/day1/README.md:22]: Please correct this.

File: docs/journey/day2/README.md [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+* [gitops environment](../../commands/environment)
+* [gitops service](../../commands/service)
+* [gitops webhook](../../commands/webhook)
+
+
+## Prerequisites
+
+* A GitOps system that has been bootstrapped in [Day 1 Operations](../day1) (Also see Day 1 Operation Prerequisities)
+* A new Git repository to be used as the new Service's source repository. 
+* Download official [gitops](../../commands/bin) binary
```
Comment (wtam2018) [inline docs/journey/day2/README.md:16]: Please fix

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+### Reference
+
+The command reference to generate and manage the above can be found [here](./commands).
+
+This project provides the tooling to generate the recommended directories and manifests as per the [Application Pipelines Model](./model).
+
+### Sample
+
+A sample GitOps repository can be found [here](https://github.com/rhd-gitops-example/gitops).
```
Comment (wtam2018) [inline docs/README.md:42]: Should we update this sample repo?  It is 2 months old.

Comment (gaganhegde) [reply docs/README.md:42]: Yes William !! can create a new task to make changes to this repo ? should we also move that to red hat developer org? @wtam2018

Comment (wtam2018) [reply docs/README.md:42]: Good point.   I copied the old sample gitops repo to https://github.com/redhat-developer/gitops-repo-example.  Please update all references to point to the new example gitops repo location.   

You can update the content of the new example repo later and use this https://issues.redhat.com/browse/GITOPS-290.   Don't need to create a new Jira.   Thanks.

Comment (wtam2018) [reply docs/README.md:42]: You should scan the document.   There should be any references to gitops-repo-example anymore.

**Comment by bigkevmcd** [comment]
I'm merging this as I want to use it as a base to tidy up all the user-facing text.

**Pull request merged**